### ISSUE TITLE
Standard translation for commonly used terms.

### ISSUE THREAD

**Issue by RobinSeaside** [lzjqsdd/scikit-learn-doc-cn#32]
1. Append the your terms in the comment;
2. Discuss controversial translation routinely;
3.  Summarize the list of terms routinely;

**Comment by RobinSeaside** [comment]
training set: 训练集
test set: 测试集
examples: 样例
bias: 偏移
variance: 方差
